package.json

```diff
@@ -60,6 +60,7 @@
     "dataloader": "^1.4.0",
     "deepmerge": "^2.1.1",
     "express": "^4.16.3",
+    "ez-ens": "^1.0.4",
```

Comment:
Curious as to why this choice of library?

Reply:
compact + it works. Seemed like a good option.